jsonschema2pojo-ant/src/main/java/org/jsonschema2pojo/ant/Jsonschema2PojoTask.java

```diff
@@ -140,6 +140,8 @@
     private String timeType = null;
 
     private String dateType = null;
+    
+    private boolean formatDateTime = false;
```

Comment:
Could you rename this config option to `formatDateTimes`? (plural) This fits better with the conventions in this project and also describes this feature better I think.